torch/csrc/distributed/autograd/engine/dist_engine.cpp

```diff
@@ -344,8 +375,7 @@ std::shared_ptr<rpc::FutureMessage> DistEngine::runEngineAndAccumulateGradients(
   auto graphTask = autogradContext->retrieveGraphTask();
   at::launch([this, graphTask, graphRoot, incrementOutstandingTasks]() {
     execute_graph_task_until_ready_queue_empty(
-        /*graph_task*/ graphTask,
-        /*root_to_execute*/ graphRoot,
+        NodeTask(graphTask, graphRoot, InputBuffer(0)),
```

Comment:
nit: missing comment about which argument it is to be consistent with next line.
Same for all call sites.